src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Circle implements Figure {
```

Comment:
The Circle class should not start with an empty line after the class declaration.